pennylane/operation.py

```diff
@@ -561,7 +561,7 @@ def compute_matrix(*params, **hyperparams):  # pylint:disable=unused-argument
         The canonical matrix is the textbook matrix representation that does not consider wires.
         Implicitly, this assumes that the wires of the operator correspond to the global wire order.
 
-        .. seealso:: :meth:`~.CNOT.matrix`
+        .. seealso:: :meth:`~.Operator.matrix`
```

Comment:
```suggestion
        .. seealso:: :meth:`~.Operator.get_matrix` and :func:`~.matrix`
```

Right now `matrix` is a property that will be deprecated, but there is a `qml.matrix` operator transform.

Reply:
Oh right, that's why the link wasn't being generated 👍